### PR TITLE
Move certificate loading function where it can be shared.

### PR DESCRIPTION
**Pull request by jcbsmpsn**

### CODE REVIEW

**Comment by jcbsmpsn** [comment]
@mikedanese

**Comment by cblecker** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/40612)*

File: cmd/kubelet/app/bootstrap.go [hunk 1]
```diff
@@ -74,7 +73,7 @@ func bootstrapClientCert(kubeconfigPath string, bootstrapPath string, certDir st
 	if err != nil {
 		return fmt.Errorf("unable to build bootstrap key path: %v", err)
 	}
-	keyData, generatedKeyFile, err := loadOrGenerateKeyFile(keyPath)
+	keyData, generatedKeyFile, err := certutil.LoadOrGenerateKeyFile(keyPath)
```
Comment (zhouhaibing089) [inline cmd/kubelet/app/bootstrap.go]: You probably need to update the `BUILD` file. :)

Comment (jcbsmpsn) [inline cmd/kubelet/app/bootstrap.go]: I'm not clear what you mean. `cmd/kubelet/app/BUILD` already has this line: 

```
        "//vendor:k8s.io/client-go/util/cert",
```

and `staging/src/k8s.io/client-go/util/cert/io.go` appears to have all the imports needed by the new function.

Comment (mikedanese) [inline cmd/kubelet/app/bootstrap.go]: If the tests pass, then you should be fine.

**Comment by jcbsmpsn** [comment]
cc @dchen1107

**Comment by dchen1107** [comment]
/lgtm 
and
/approve

**Comment by mikedanese** [comment]
@k8s-bot bazel test this

**Comment by mikedanese** [comment]
@k8s-bot unit test this

**Pull request merged**